source/system/Numbers.ooc

```diff
@@ -137,6 +137,7 @@ SizeT: cover from size_t extends ULLong
 SSizeT: cover from ssize_t extends LLong {
 	toString: func -> String { "%u" formatSSizeT(this) }
 }
+PtrDiff: cover from ptrdiff_t extends SSizeT
```

Comment:
Ah - easily confusing since `sizeof(C) == sizeof(Small)` is not only valid ooc, but also true. :)
Anyway, I took the definition of `PtrDiff` as it appears in the original sdk. I don't understand why you assume ooc to be non-confusing.